### PR TITLE
Manual buckets not required before backend registration for all cloud…

### PR DESCRIPTION
**Pull request by ampawar30**
In previous releases, Manual buckets were required for backend registration. Now from Isebela release onwards, buckets cretion is fully atomated from SODA.

Manual buckets not required before backend registration for all cloud vendors now. List need updation
Screenshot from 2021-09-29 16-21-01

### CODE REVIEW

File: content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md [hunk 1]
```diff
@@ -79,15 +79,10 @@ Click on (+) for registering a storage backend
 
 and then provide valid region, endpoint, Access and Secret Key parameters
 
-##### Note:
-For cloud vendors like AWS, Azure, Huawei and GCP remote cloud buckets need not be created manually by user prior to backend registration. Remote cloud buckets can be created from SODA for these cloud vendors
+##### Register a cloud backends as follows
```
Comment (anvithks) [inline content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md:82]: nit: Change the heading level to `###` to match the subsequent heading levels.
For ex: The next heading is  `### Create bucket and upload the object`.

**Review by anvithks — APPROVED** [review]
LGTM.
@PravinRanjan10 Please verify if the contents are right and the instruction of creating cloud bucket is not needed for any of our supported cloud vendors.

Thank you @ampawar30 for you contribution
I am changing the destination branch to the `soda-hackathon-2021` branch.
Please ensure that all further PRs are raised to the above branch.

**Review by PravinRanjan10 — APPROVED** [review]
LGTM. 
Please address @anvithks comments.

**Review by anvithks — CHANGES_REQUESTED** [review]
Please address the review comments

**Review by anvithks — APPROVED** [review]
LGTM

Thank you @ampawar30 !

**Pull request merged**